doc/configcommands-linked.asciidoc

```diff
@@ -0,0 +1,1620 @@
+'''
```

Comment:
Since this file is auto-generated, let's remove it from the repository and add it to the .gitignore.